### ISSUE TITLE
the weight of videoswin-l

### ISSUE THREAD

**Issue by srrn-lemon** [tsujuifu/pytorch_empirical-mvm#9]
Hi,

  Thank you for your insightful paper. 
  I noticed that in Table 1, the TVF employs the VidSwin-L model, but your settings only utilize the VidSwin-B model. 
```
if "3d_feature" in args.mvm_target:
            from visbackbone.video_swin import get_vidswin_model
            feature_args = copy.deepcopy(args)
            feature_args["vis_backbone_init"] = "3d"
            feature_args["vis_backbone_size"] = "base"
            feature_args["kinetics"] = 600
            self.feature_model = get_vidswin_model(feature_args)
            feat_size = self.feature_model.norm.normalized_shape[0]
            LOGGER.info("MVM-Feature: MVM with 3d feature value from video swin as gt")
            self.fc_mvm = T.nn.Sequential(*[T.nn.Dropout(0.1), T.nn.Linear(self.hidden_size, self.hidden_size*2), T.nn.ReLU(inplace=True), 
                                            T.nn.Linear(self.hidden_size*2, feat_size)])
```
  However, in Table 15, when using the VidSwin-B model, the R1 metric decreases by 7%. Could you please provide the weights for the VidSwin-L model and the corresponding setting?

Thank you!
